### PR TITLE
[math] Add a new math method: multiply.

### PR DESCRIPTION
**Pull request by Skyge**

### CODE REVIEW

File: src/strategies/math/options.ts [hunk 1]
```diff
@@ -3,6 +3,7 @@ export type Operand = StrategyOperand | ConstantOperand;
 export interface Options {
   operands: Operand[];
   operation: Operation;
+  multiplier: number;
```
Comment (ChaituVR) [inline src/strategies/math/options.ts:6]: Hi @Skyge It should be like other types, you can pass multiplier into operands like other operations

**Review by ChaituVR — CHANGES_REQUESTED** [review]
Also would be great if you can change Readme as well

File: src/strategies/math/options.ts (hunk 1 above)
Comment (Skyge) [reply src/strategies/math/options.ts:6]: Okay, let me update it.

Comment (Skyge) [reply src/strategies/math/options.ts:6]: So do you mean like this:
```
"operands": [
          {
            "type": "strategy",
            "strategy": {
              "network": "1",
              "name": "erc20-balance-of",
              "params": {
                "address": "0x6b175474e89094c44da98b954eedeac495271d0f",
                "symbol": "DAI",
                "decimals": 18
              }
            },
            "multiplier": 6
          }
        ],
```

**Comment by Skyge** [comment]
It seems like there is something wrong.

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**